docs/App.jsx

```diff
@@ -142,8 +163,9 @@ export default class App extends Component {
 				TextFieldRule,
 				DateField,
 				CopyableList,
+				Toggle,
 				...Icons,
-			}
+			};
 
 			if (props.metaString == null) {
```

Comment:
I did not take too much care on the "UI web" because it is not relevant to other repos... it is just like a web to show the components we have... maybe it will be removed in favour of Storybook some day 🤷‍♂️ I think it does not make sense to spend too much time refactoring it. What do u think?